markdown/org/docs/patterns/waralee/instructions/en.md

```diff
@@ -1,54 +1,73 @@
+</Fixme>
 
-##### If you do want pockets, make them first
-If you're going to make the pockets, make them before putting one piece of fabric on the other. It's easier to work with the parts still separated.
+#### Front Pockets
 
-</Note>
+Okay, the facing is used twice (per pocket). The pocket itself (part 3) is cut-on-fold.
+
+You attach one facing to the pocket after folding the sides under (for finishing), align with the slanted side (or add a small SA, since you will be needing to sew this to the other side of the pocket, and that seam will need to be finished. I use French seams for my pockets, so reducing bulk is good). This will be the 'inside' of the pocket.
+
+Iron-on some interfacing on the wrong side of the 'pants' fabric. about 3cm larger than the pocket opening (I use pinking shears to cut this, so it's less obvious from the right side that the interfacing is there).
 
-If desired, sew the pockets.
+Lay the other facing part on the right side of the 'pants', with most of the fabric towards the crotch. You want about 1/4", or 6mm, sticking out on the side-seam side.
+Lay the 'inside' of the pocket (the side without the already attached facing) on the right side of the 'pants'. Add another 2cm or so for the seam and French finish as mentioned above. Stitch the two lines of stitching as you would on any welt pocket.
 
-I included placements for the front pockets that will be hidden by the flaps in the front. But they will still be easily accessible to put your phone or keys in. These are single welt pockets. The ones in the front are at an angle and the ones in the back are horizontally oriented. The pocket templates should allow for enough fabric to attach the pocket to the waistband. If you want particularly deep pockets, please adjust accordingly.
+Cut the pocket as usual, and push everything to the wrong side. Check to make sure you have the inside/outside and facing correctly. (I cheat a lot with this step by attaching that piece of facing after I do this step. If I get the inside/outside wrong, no harm done). Press this well. I press open both seams before folding them back onto themselves. Now take the longer part of the facing you just cut the opening through and fold it like a single welt. Stitch it in place by sewing through the seam closest to the crotch. Fold under the last part of it, and sew that to the pocket. Sew the little triangles on the ends of the pocket to the facing and pocket.
+Now bring the whole thing back to the outside, and sew a seam around it, about 5mm from the edge. Push it back inside, press and stitch 6mm from the edge to complete the French seam.
 
-## Step 2: Hem the sides and bottom
+As a last step, I sew the side closest to the side through all layers. This anchors the pocket to that side well.
+
+Make sure you sew the top of the pocket into the waist hem.
+
+#### Back Pockets
```

Comment:
Is this the bit that the fixme above is referring to? It feels strange to have a heading with no body. What do you think about moving the fixme to this section so it is clear why it is missing?